ocs_ci/ocs/constants.py

```diff
@@ -1109,15 +1109,20 @@
 SCALE_LABEL = "scale-label=app-scale"
 # TODO: Revisit the dict value once there is change in instance/vm/server type
 # TODO: Generic worker count value to support all kind of pods.
-# Note: Below worker count value is based on nginx pod
-# aws dict value is based on the manual execution result with m5.4xlarge instance and nginx pod
+# Note: Below worker count value is based on performance pod and each pod
+# will be attached with 20 PVC's each, i.e. if we create 3000 PVCs then
+# will be creating 150 pods each pod attached with 20 PVCs and in PVC
+# there will be minimal fio started
+# aws dict value is based on the manual execution result with m5.4xlarge instance and perf pod
 # vmware dict value is based on each worker vm config of min 12CPU and 64G RAM
 # bm dict value is based on each worker BM machine of config 40CPU and 256G/184G RAM
 # azure dict value is based on assumption similar to vmware vms min worker config of 12CPU and 64G RAM
 SCALE_WORKER_DICT = {
-    1500: {"aws": 12, "vmware": 15, "bm": 5, "azure": 15},
-    3000: {"aws": 24, "vmware": 30, "bm": 10, "azure": 30},
-    4500: {"aws": 36, "vmware": 45, "bm": 15, "azure": 45},
+    1500: {"aws": 3, "vmware": 3, "bm": 2, "azure": 3},
```

Comment:
line 1121 values are required app worker nodes in the cluster for running app pods.
As you mentioned we should explore this on how to run 60% IO, for now we have to create separate app pods. i.e. 1 or 2 pod with bigger PVC size and fill up the cluster separately, again this have to be added separately for more controll over IO and POD, with kube_job way of pod creations this will be difficult to achieve.